packages/dds/pact-map/.eslintrc.js

```diff
@@ -18,7 +18,7 @@ module.exports = {
 			files: ["*.spec.ts", "src/test/**"],
 			rules: {
 				// Test files are run in node only so additional node libraries can be used.
-				"import/no-nodejs-modules": ["error", { allow: ["assert", "events"] }],
+				"import/no-nodejs-modules": ["error", { allow: ["node:assert", "events"] }],
```

Comment:
Nit: the tests don't seem to import from the `events` package, so we could maybe remove that second bit here.

Reply:
```suggestion
				"import/no-nodejs-modules": ["error", { allow: ["node:assert"] }],
```